src/bssl.rs

```diff
@@ -38,6 +38,7 @@ macro_rules! bssl_test {
             }
 
             init::init_once();
+            ::std::env::set_current_dir(::test::ring_src_path()).unwrap();
```

Comment:
I'm not quite sure why this is needed on iOS but not Android. Here's how we do it on Android; can we do the same thing on iOS?
https://github.com/briansmith/ring/blob/c84e4118f8a956b306aa7c8a3eafa23159f05426/mk/travis.sh#L82-L90

Also, why is it important to switch the current directory, instead of just changing file accesses to use the full path?